examples/sample_token_generate_refresh.py

```diff
@@ -0,0 +1,58 @@
+import sys
+
+from uid2_client import Uid2PublisherClient
+from uid2_client import TokenGenerateResponse
+from uid2_client import TokenGenerateInput
+
+
+def _usage():
+    print('Usage: python3 sample_token_generate_refresh.py <base_url> <auth_key> <secret_key>', file=sys.stderr)
+    sys.exit(1)
+
+
+if len(sys.argv) <= 3:
+    _usage()
+
+base_url = sys.argv[1]
+auth_key = sys.argv[2]
+secret_key = sys.argv[3]
+
+
+publisher_client = Uid2PublisherClient(base_url, auth_key, secret_key)
+print("Generating Token")
+token_generate_response = publisher_client.generate_token(TokenGenerateInput.from_email("test@email.com").do_not_generate_tokens_for_opted_out())
```

Comment:
dunno if it's wild coincidence, test@email.com seemed to have opted out of uid2 so the status stated opted out and you won't get an identity in the response. maybe use some obscure email like testpythonsdksampletokengenerate@email.com to make sure it's not opted out.

maybe add some codes to check the token_generate_response.status and print and exit gracefully if the email is opted out.

Make sure you try running these sample scripts. @cody-constine-ttd will be able to suggest how to find the url and keys to use.

Reply:
I made reference to this in UID2-1224

Reply:
changed the email and it generates an identity for me in integ